### PR TITLE
WeAreSorryHubBase - bringing `HubBase` back to fix binary compatibility

### PR DESCRIPTION
**Pull request by moozzyk**
#3489

### CODE REVIEW

**Comment by moozzyk** [comment]
CC: @halter73

**Comment by moozzyk** [comment]
Ping @halter73 @davidfowl

File: src/Microsoft.AspNet.SignalR.Core/Hub.cs [hunk 1]
```diff
@@ -8,68 +8,18 @@ namespace Microsoft.AspNet.SignalR
...
-        protected virtual void Dispose(bool disposing)
-        {
-        }
-
-        public void Dispose()
-        {
-            Dispose(true);
+            get
+            {
+                return HubClients;
```
Comment (halter73) [inline src/Microsoft.AspNet.SignalR.Core/Hub.cs]: Glad we're not reintroducing that stackoverflow bug :grinning:

**Comment by halter73** [comment]
:ship:

File: src/Microsoft.AspNet.SignalR.Core/Hub.cs (hunk 1 above)
Comment (moozzyk) [inline src/Microsoft.AspNet.SignalR.Core/Hub.cs]: Yeah. One of the goals was to keep the fix :)

**Pull request closed without merging**